browser/brave_profile_prefs.cc

```diff
@@ -120,6 +120,9 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
 
   // IPFS companion extension
   registry->RegisterBooleanPref(kIPFSCompanionEnabled, false);
+
+  // Dapp detection
+  registry->RegisterBooleanPref(kDappDetectionEnabled, false);
```

Comment:
I set false as a default.

Reply:
This preference is no longer used so it was removed.